src/pages/IndividualFundraiserPage.js

```diff
@@ -1,26 +1,69 @@
 import React, {useState, useEffect} from 'react'
 import {  useParams } from "react-router-dom";
-import placeholderImage from '../imgs/Fundraising-Tree.jpg'
+import placeholderImage from '../imgs/placeHolderFundraiserPic.jpg'
 
 const IndividualFundraiserPage = () => {
-    // const [fundraiserRequestOptions, setFundraiserRequestOptions] = useState({});
     const [fundraiser, setFundraiser] = useState([]);
+    const [user, setUser] = useState([]);
     const [tags, setTags] = useState([]);
-    const {fundraiserId} = useParams();
-
-    // Set the request url
-    const url = "https://localhost:7000/api/fundraiser/".concat(fundraiserId);
+    const {userId, fundraiserId} = useParams();
+    
+    // Set the fundraiser and user urls
+    const userUrl = "https://localhost:7000/api/user/".concat(userId);
+    const fundraiserUrl = "https://localhost:7000/api/fundraiser/".concat(fundraiserId);
 
```

Comment:
Instead of concat try using `https://localhost:7000/api/fundraiser/${fundraiserId}`

(It basically does the same thing but looks cleaner